README.md

```diff
@@ -8,132 +8,92 @@
+The tests for this project are written in [minitest](https://github.com/seattlerb/minitest), a testing framework by Seattle Ruby Brigade.
+
+Do not move onto a new wave of requirements until the minimum requirements of the previous wave are complete and your specs are green across the board.
+
+Here is what it looks like when all 16 tests are passing:
+![all 16 tests are passing message](assets/passing-tests-screenshot.png)
 
 ### Pair Programming
 Utilize good pair programming practices. Refer to articles from the [Agile Alliance](http://guide.agilealliance.org/guide/pairing.html), the [Agile Institute](http://powersoftwo.agileinstitute.com/2015/02/benefits-of-pair-programming-revisited.html), and [our own suggestions for pairing](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/programming-techniques-pairs-or-solo.md#pair-programming) if you need a refresher for some best practices. Switch _driver_ and _navigator_ roles often. When there is uncertainty or confusion, step away from the keyboard and discuss, plan, and document on paper or whiteboard before continuing.
 
-## Baseline
-### Setup
+### Driver Code
+We have provided some driver code for your Adagrams game in the files `wave-1-game.rb`, `wave-2-game.rb`, `wave-3-game.rb`, and `wave-4-game.rb`. Running `$ ruby wave-1-game.rb` will begin a command-line game that uses your Adagrams code. The boilerplate code will break the first time you run it: working through the waves specified below should create a running version of the game. **Implementing code to make this game run is not a substitute for making the tests pass**. It is simply there for you and your pair to reference how the Game may run during each wave, to have better perspective of what your program can do, and to get some practice reading other peoples' code. We fully expect you to create the specified methods to specification and making the tests pass.
+
+## Setup Requirements
+### Setup For Collaboration
 1. You'll be working with an assigned pair. High-five your pair.
 1. Choose **one person** to fork the project repo.
 1. Add the other person in the pair (who didn't fork) to the forked repo as a _collaborator_. Instructions [here](https://help.github.com/articles/inviting-collaborators-to-a-personal-repository/).
 1. Both individuals will clone the forked repo: `$ git clone [YOUR FORKED REPO URL]`
```

Comment:
If both team members are cloning are we going to need to teach `git pull` before the project?  I would suggest having them pick a single machine to work from.

Reply:
I think the plan is to have them work on a single machine to start, then we teach pull/push on Wednesday.

Reply:
I'm going to adjust the instructions to state that in the beginning of the project, one machine will be used, and later on in the week, we'll learn how to collaborate with Git